tests/test_integrated_channels/test_integrated_channel/test_exporters/test_content_metadata.py

```diff
@@ -167,6 +168,80 @@ def test_content_exporter_failed_create_export(self, mock_get_catalog_diff, mock
         assert not update_payload
         assert not delete_payload
 
+    @mock.patch('integrated_channels.integrated_channel.exporters.content_metadata.EnterpriseCatalogApiClient')
+    def test_content_exporter_truncation_bug_export(self, mock_api_client):
```

Comment:
this test replicates the behavior we were seeing in production. it fails with the old order of truncate operations.